flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
didn't tested for other platforms

Reply:
it's clear the python code will not be tested that way.